src/neptune_scale/__init__.py

```diff
@@ -277,63 +277,89 @@ def _create_run(
             create=CreateRun(
                 family=self._family,
                 fork_point=fork_point,
-                experiment_id=as_experiment,
+                experiment_id=experiment_name,
                 creation_time=None if creation_time is None else datetime_to_proto(creation_time),
             ),
         )
         self._operations_queue.enqueue(operation=operation)
 
+    def log_metrics(
+        self,
+        step: Optional[Union[float, int]] = None,
+        timestamp: Optional[datetime] = None,
+        data: Optional[Dict[str, float]] = None,
+    ) -> None:
+        self.log(step=step, timestamp=timestamp, metrics=data)
+
+    def log_fields(
```

Comment:
`fields` is still bad. Even worse given that "string set" is a type of field, too.